### ISSUE TITLE
Can't add a Car 

### ISSUE THREAD

**Issue by dirks67** [Farfar/homeassistant-seatconnect#65]
Hi,

I'll try to add a car to the integration (Cupra Leon ST PHEV). After entering Mail an Password the dialog does not close. In the Log I found "2024-07-29 15:49:54.555 WARNING (MainThread) [homeassistant.data_entry_flow] custom_components.seatconnect.config_flow::SeatConnectConfigFlow calls async_show_progress without passing a progress task, this is not valid and will break in Home Assistant Core 2024.8. Please report it to the custom integration author"

**Comment by kRew94** [comment]
I have the exact same problem. The integration needs to be updated to use the progress_task argument instead of the step_id which is deprecated since 2024.8.

**Comment by federicotravaini** [comment]
I have a similar problem with a CUPRA FORMENTOR: I had the integration working fine till a couple of days ago, then I had to delete and re-enable the integration and now is not working anymore. I tried to delete completely and configure it from scratch new but after I enter user, password and pin, it just doesn't load the device/entities (even if the configuration looks like is correctly set up)

**Comment by Fabhino22Elz** [comment]
I do have the same issue. The Login Mask shows up, and then the spinner is running and nothing happens.

@Farfar is there a possibility to get support on this?

**Comment by MantovaniMatteo** [comment]
> I have a similar problem with a CUPRA FORMENTOR: I had the integration working fine till a couple of days ago, then I had to delete and re-enable the integration and now is not working anymore. I tried to delete completely and configure it from scratch new but after I enter user, password and pin, it just doesn't load the device/entities (even if the configuration looks like is correctly set up)

Hi Federico,
I have a Cupra Formentor Plug-in.
I try to use this integration https://github.com/Farfar/homeassistant-seatconnect to see all data from my Car.

But after 4/5 hours the integration disconnects with No vehicles found and then **MyCupra App don't see anythink.**
No information is available in the myCupra app with several errors if I try to see permissions in the app.

The integration had 60 seconds to refresh the data.

Someone have any solution?

**Comment by federicotravaini** [comment]
> > I have a similar problem with a CUPRA FORMENTOR: I had the integration working fine till a couple of days ago, then I had to delete and re-enable the integration and now is not working anymore. I tried to delete completely and configure it from scratch new but after I enter user, password and pin, it just doesn't load the device/entities (even if the configuration looks like is correctly set up)
> 
> Hi Federico, I have a Cupra Formentor Plug-in. I try to use this integration https://github.com/Farfar/homeassistant-seatconnect to see all data from my Car.
> 
> But after 4/5 hours the integration disconnects with No vehicles found and then **MyCupra App don't see anythink.** No information is available in the myCupra app with several errors if I try to see permissions in the app.
> 
> The integration had 60 seconds to refresh the data.
> 
> Someone have any solution?

Hi, 300 seconds is the minimum required not to end all the token before midnight

**Comment by Fabhino22Elz** [comment]
> > > I have a similar problem with a CUPRA FORMENTOR: I had the integration working fine till a couple of days ago, then I had to delete and re-enable the integration and now is not working anymore. I tried to delete completely and configure it from scratch new but after I enter user, password and pin, it just doesn't load the device/entities (even if the configuration looks like is correctly set up)
> > 
> > 
> > Hi Federico, I have a Cupra Formentor Plug-in. I try to use this integration https://github.com/Farfar/homeassistant-seatconnect to see all data from my Car.
> > But after 4/5 hours the integration disconnects with No vehicles found and then **MyCupra App don't see anythink.** No information is available in the myCupra app with several errors if I try to see permissions in the app.
> > The integration had 60 seconds to refresh the data.
> > Someone have any solution?
> 
> Hi, 300 seconds is the minimum required not to end all the token before midnight

How do you guys even manage to sing in to the App?

I do have a CUPRA Formentor Plugin from 2024 and if i try to Login it is not working.

I enter my email and password and the Login just keeps Showing the loading Spinner.

Anyone here to help with that?

Greetings

**Comment by MantovaniMatteo** [comment]
Also for me Cupra integration didn't work. Use Sea Connect integration from Farfar and change refresh rate to 300.

**Comment by Fabhino22Elz** [comment]
> Also for me Cupra integration didn't work. Use Sea Connect integration from Farfar and change refresh rate to 300.

So you also do have a formentor?

And you are able to Login? I also use the Seat App but i cant Login. The Spinner After Login and password Never stops.

I just cant Login anyhow.

are there and additional Settings that Need to be made before Login? Wäre do you Chance the Refresh rate?

**Comment by Farfar** [comment]
This code is no longer being maintained by me and requires contributors in order to fix issues.

**Comment by MantovaniMatteo** [comment]
> > Also for me Cupra integration didn't work. Use Sea Connect integration from Farfar and change refresh rate to 300.
> 
> So you also do have a formentor?
> 
> And you are able to Login? I also use the Seat App but i cant Login. The Spinner After Login and password Never stops.
> 
> I just cant Login anyhow.
> 
> are there and additional Settings that Need to be made before Login? Wäre do you Chance the Refresh rate?

Yes, I have Cupra Formentor 204 plug-in. Normally I use MyCupra app but you can try to use SEAT CONNECT android app and see if you have the same problem to login. Must works! Then try also to login on webpage and see if is necessary confirm some authorization requests:
https://cupraid.vwgroup.io

When you add Seat Connect Integration, inside CONFIGURE you can see Scan_Interval.
